coalib/results/Result.py

```diff
@@ -208,8 +208,8 @@ def apply(self, file_dict: dict):
         :param file_dict: A dictionary containing all files with filename as
```

Comment:
this time commit message does really need change, your shortlog is to general^^

You did simplify code, so you could just say `Result: Simplify` or `Result.apply(): Simplify`.

\+ The issue reference `Fixes xxx` is wrong, please remove it^^ The other commit does fix the issue :+1:

\+
> [...] and it follows the param definition of diffs.

I don't understand that :O